pkgs/by-name/dr/dragon-drop/package.nix

```
@@ -20,16 +20,20 @@ stdenv.mkDerivation (finalAttrs: {
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [ gtk3 ];
 
+  makeFlags = [ "NAME=dragon-drop" ];
   installFlags = [ "PREFIX=${placeholder "out"}" ];
   postInstall = ''
-    ln -s $out/bin/dragon $out/bin/xdragon
+    ln -s $out/bin/dragon-drop $out/bin/xdragon
   '';
 
   meta = with lib; {
     description = "Simple drag-and-drop source/sink for X or Wayland (called dragon in upstream)";
     homepage = "https://github.com/mwh/dragon";
-    license = licenses.gpl3;
-    maintainers = with maintainers; [ das_j ];
-    mainProgram = "xdragon";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [
```

```suggestion
    license = lib.licenses.gpl3Plus;
    maintainers = with lib.maintainers; [
```
We can remove `with lib;` at the same time. See https://github.com/NixOS/nixpkgs/issues/292468#issuecomment-2016454242